recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,238 @@
+        cmake.definitions["UA_ENABLE_DISCOVERY"] = self.options.discovery
+        cmake.definitions["UA_ENABLE_DISCOVERY_MULTICAST"] = self.options.discovery_multicast
+        cmake.definitions["UA_ENABLE_DISCOVERY_SEMAPHORE"] = self.options.discovery_semaphore
+        cmake.definitions["UA_ENABLE_QUERY"] = self.options.query
+        cmake.definitions["UA_ENABLE_ENCRYPTION"] = self.options.encription
+        cmake.definitions["UA_ENABLE_JSON_ENCODING"] = self.options.json_support
+        if self.options.pub_sub != "None":
+            cmake.definitions["UA_ENABLE_PUBSUB"] = True
+            if self.settings.os == "Linux" and self.options.pub_sub == "Ethernet":
+                cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP"] = True
+            elif self.settings.os == "Linux" and self.options.pub_sub == "Ethernet_XDP":
+                cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP_XDP"] = True
+            else:
+                print("PubSub over Ethernet is not supported for your OS!")
+        cmake.definitions["UA_ENABLE_DA"] = self.options.data_access
+        if self.options.compiled_nodeset_descriptions == True:
+            cmake.definitions["UA_ENABLE_NODESET_COMPILER_DESCRIPTIONS"] = self.options.compiled_nodeset_descriptions
+            cmake.definitions["UA_NAMESPACE_ZERO"] = "FULL"
+        else:
+            cmake.definitions["UA_NAMESPACE_ZERO"] = self.options.namescpae_zero
+        cmake.definitions["UA_ENABLE_MICRO_EMB_DEV_PROFILE"] = self.options.embedded_profile
+        cmake.definitions["UA_ENABLE_TYPENAMES"] = self.options.typenames
+        cmake.definitions["UA_ENABLE_STATUSCODE_DESCRIPTIONS"] = self.options.readable_statuscodes
+        cmake.definitions["UA_ENABLE_HARDENING"] = self.options.hardening
+        if self.settings.compiler == "Visual Studio" and self.options.shared == True:
+            cmake.definitions["UA_MSVC_FORCE_STATIC_CRT"] = True
+        if version > "1.0.1":
+            cmake.definitions["UA_COMPILE_AS_CXX"] = self.options.cpp_compatible
+        cmake.configure()
+        return cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE-CC0", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "open62541"
+        self.cpp_info.names["cmake_find_package_multi"] = "open62541"
+        self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.includedirs = [
+            "include",
+            os.path.join("include", "plugin")
+        ]
+        if self.options.single_header:
+            self.cpp_info.defines.append("UA_ENABLE_AMALGAMATION")
+        if self.settings.os == "Windows":
+            self.cpp_info.libs.append("ws2_32")
```

Comment:
system_libs

Reply:
Resolved in the next commit, just running local tests